src/popup/router/routes.js

```diff
@@ -436,11 +436,28 @@ export default [
   },
   {
     name: 'token-details',
-    path: '/balances/:id',
+    path: '/balancesx/:id',
```

Comment:
`balancesx` in my opinion is meaningless. I suggest to make the path closer to the name.
Eg.:
`/balances/token/:id`
`/balances-token/:id`